doc/classes/BaseMaterial3D.xml

```diff
@@ -582,7 +582,7 @@
 			Default cull mode. The back of the object is culled when not visible.
 		</constant>
 		<constant name="CULL_FRONT" value="1" enum="CullMode">
-			The front of the object is culled when not visible.
+			The front of the object is culled when visible.
```

Comment:
```suggestion
			Front face triangles will be culled when facing the camera. This results in only the back side of triangles being drawn. For closed-surface meshes this means that the interior of the mesh will be drawn instead of the exterior.
```